app/views/admin/resource/_index.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= link_to "New #{resource_class}", send("new_admin_#{resource_as_sym}_path") %>
+
+<table>
+  <tr>
+    <th>Name</th>
+    <th>Actions</th>
+  </tr>
+  <% collection.each do |resource| %>
+    <tr>
+      <td><%= link_to resource.name, send("admin_#{resource_as_sym}_path", resource) %></td>
```

Comment:
not quite there yet @damwhit @erinnachen every resource is not going to have a "name" attribute, nor just one attribute we are listing. we want a way to configure the attributes we care about showing and having iterated over in the form. similar to how we use `whitelist` in the controller, but specific to the display fields.